kartograf/merge.py

```diff
@@ -67,46 +112,57 @@ def general_merge(base_file, extra_file, extra_filtered_file, out_file, silent=F
             extra_nets_int.append(netw_int)
             extra_asns.append(asn.strip())
             extra_pfxs.append(pfx)
+            if ipn.version == 4:
+                root_net = int(pfx.split(".")[0])
+            else:
+                root_net = str(pfx).split(":")[0]
+            extra_pfxs_leading.append(root_net)
 
     df_extra = pd.DataFrame()
```

Comment:
Seems to me like with this approach we don't really need pandas either? At least I think it's usage doesn't bring any performance improvements, right? I guess the csv print function is kind of handy but if it's only that I think would lean towards getting rid of another dependency.

Reply:
we use the pandas dataframe for storing the extra networks and iterating through them with `itertuples`, which is nice, and filtering out the included, and then the csv. We also use it in `coverage.py`. Definitely possible to remove it but I'd like to do that in a follow-up PR since it touches a handful of places/logic, if that's ok.